test/legacy_test/test_batch_norm_op_v3.py

```diff
@@ -0,0 +1,524 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+from eager_op_test import (
+    OpTest,
+    _set_use_system_allocator,
+    convert_float_to_uint16,
+)
+
+import paddle
+import paddle.nn.functional as F
+from paddle.fluid import core
+
+paddle.enable_static()
+
+np.random.seed(123)
+paddle.seed(123)
+
+_set_use_system_allocator(True)
+
+
+def batch_norm_wrapper(
+    x,
+    running_mean,
+    running_variance,
+    weight,
+    bias,
+    is_test,
+    momentum,
+    epsilon,
+    data_format,
+    use_global_stats,
+):
+    # breakpoint()
+    y = F.batch_norm(
+        x,
+        running_mean,
+        running_variance,
+        weight,
+        bias,
+        training=not is_test,
+        momentum=momentum,
+        epsilon=epsilon,
+        data_format=data_format,
+        use_global_stats=use_global_stats,
+    )
+    z = F.relu(y)
+    return z
+
+
+class TestBatchNormOp(OpTest):
+    def setUp(self):
+        self.python_api = batch_norm_wrapper
+        self.public_python_api = batch_norm_wrapper
+        self.op_type = "batch_norm"
+        self.prim_op_type = "comp"
+        self.python_out_sig = ["Y"]
+        self.initConfig()
+        self.initTestCase()
+
+    def test_check_output(self):
+        if self.dtype not in ("uint16", "float16"):
+            self.check_output_with_place(
+                core.CPUPlace(),
+                no_check_set=None,
+                atol=self.fw_comp_atol,
```

Comment:
atol and rtol should be set in setUp function for prim check.

Reply:
done